usr/share/rear/restore/BORG/default/300_load_archives.sh

```diff
@@ -63,13 +63,15 @@ while true ; do
         LogUserOutput "[0] Show all archives again"
     fi
 
+    local abort_choice
+    (( abort_choice = archive_cache_lines_total + 1 ))
     # Show "Exit" option.
     UserOutput ""
-    LogUserOutput "[$(( archive_cache_lines_total + 1 ))]" Exit
+    LogUserOutput "[$abort_choice]" Exit
     UserOutput ""
 
     # Read user input.
-    choice="$( UserInput -I BORGBACKUP_ARCHIVE_TO_RECOVER -p "Choose archive to recover from" )"
+    choice="$( UserInput -I BORGBACKUP_ARCHIVE_TO_RECOVER -D "$abort_choice" -p "Choose archive to recover from" )"
```

Comment:
@rear/contributors here I'm not sure if aborting is really a good default choice. Would it be better to use the first archive as the default?

Reply:
Very good point, I didn't read and understand the code properly. Thanks a lot!